hack/aks/README.md

```diff
@@ -29,6 +29,8 @@ AKS Clusters
   swift-cilium-up              Bring up a SWIFT Cilium cluster
   swift-up                     Bring up a SWIFT AzCNI cluster
   windows-cniv1-up             Bring up a Windows AzCNIv1 cluster
+  dualstack-overlay-byocni-up  Brings up an dualstack overlay cluster without CNS and CNI installed
+  dualstack-windows-byocni-up: Add windows node pool to dualstack overlay cluster
```

Comment:
I don't believe there is anything dualstack specific about the windows nodepool. Lets change this to `windows-nodepool-up`